docs/development/oracle_jdbc.md

```diff
@@ -0,0 +1,16 @@
+# Oracle Maven Access
+* Orion bundles Oracle JDBC driver which is downloaded from Oracle Maven repository by Gradle.
+* You need to provide your Oracle account credentials to access Oracle Maven repository.
```

Comment:
The reading of this line suggests this PR is introducing a dependency on having a OTN account to build Orion, is that correct?

Reply:
@CjHare  It is partially correct now that I am re-reading it. The Orion would still build without Oracle JDBC if we don't create project properties. It needs to be re-worded as `Orion can optionally bundle JDBC driver...`

Reply:
but your point is correct, this PR is introducing a dependency on having an Oracle account in order to bundle Oracle JDBC driver